examples/suit_update/tests/01-run.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import subprocess
+import sys
+import tempfile
+
+from testrunner import run
+
+# Default test over loopback interface
+COAP_HOST = "[fd00:dead:beef::1]"
+
+UPDATE_TIMEOUT = 120
+MANIFEST_TIMEOUT = 15
+
+USE_ETHOS = int(os.getenv("USE_ETHOS", "1"))
+TAP = os.getenv("TAP", "tap0")
```

Comment:
Also need to change to riot0 if my documentation changes are applied.

Reply:
done